### PR TITLE
Add Local Test Packages

### PR DESCRIPTION
**Pull request by doriac11**
# Description
For the local Acceptance/Unit tests give a path to a test package to upload so the tests can be without any manual steps (putting the package on a specific path)

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Acceptance/Unit test

### CODE REVIEW

**Review by nikitajoshi1 — APPROVED** [review]

File: powerflex/provider/protection_domain_resource_test.go [hunk 1]
```diff
@@ -293,8 +293,6 @@ func TestAccPDResource(t *testing.T) {
 				Config: ProviderConfigForTesting + updatePDRfCacheTest,
 				Check: resource.ComposeAggregateTestCheckFunc(
 					resource.TestCheckResourceAttr(resourceName, "name", "test_acc_pd_1"),
-					resource.TestCheckResourceAttr(resourceName, "rebuild_network_throttling_in_kbps", "10240"),
```
Comment (doriac11) [inline powerflex/provider/protection_domain_resource_test.go:296]: These are contradictory settings with the config in the test so I removed them.

**Comment by doriac11** [comment]
run e2e pass

**Comment by doriac11** [comment]
e2e pass

**Comment by doriac11** [comment]
run e2e tests

**Comment by shenda1** [comment]
@doriac11 Please add the execution result

**Comment by doriac11** [comment]
> @doriac11 Please add the execution result

Hey @shenda1,

This change was for the PackageResource here is the test output aginst the mock server
```
=== RUN   TestAccPackageResource
--- PASS: TestAccPackageResource (13.48s)
PASS
ok      terraform-provider-powerflex/powerflex/provider 16.567s
```

**Comment by doriac11** [comment]
run e2e test

**Review by shenda1 — APPROVED** [review]

**Review by nikitajoshi1 — APPROVED** [review]

**Comment by doriac11** [comment]
run e2e test

**Pull request merged**